client/packages/coldchain/src/Equipment/utils.ts

```diff
@@ -61,7 +61,7 @@ export const assetsToCsv = (
       Formatter.csvDateString(node.warrantyEnd),
       node.serialNumber,
       status ? t(status.key) : '',
-      node.needsReplacement,
+      node.needsReplacement ? node.needsReplacement?.toString() : 'false',
```

Comment:
Thanks Zac! Just wondering if they need to be a string. I think we can go without it and it should work the same 

`node.needsReplacement ? node.needsReplacement : false,`

Reply:
Note* `Formatter.csv` formats the value that is returned by `assetsToCsv` to a string. So thought we could let it handle the type conversion instead